posthog/models/person/backends.py

```diff
@@ -0,0 +1,19 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.backends import ModelBackend
+
+
+class CaseInsensitiveModelBackend(ModelBackend):
+    def authenticate(self, request, username=None, password=None, **kwargs):
+        UserModel = get_user_model()
+        if username is None:
+            username = kwargs.get(UserModel.USERNAME_FIELD)
+        try:
+            case_insensitive_username_field = "{}__iexact".format(UserModel.USERNAME_FIELD)
+            user = UserModel._default_manager.get(**{case_insensitive_username_field: username})
```

Comment:
Actually, looking at the original `def authenticate()` of `ModelBackend`, it seems there's an even simpler way to do this. 😅 We can simply override `def get_by_natural_key()` in `UserManager` (https://github.com/thetarunkochar/posthog/blob/master/posthog/models/user.py#L19) to do an `iexact` query  and the default `ModelBackend` will pick that up – removing the need for subclassing `ModelBackend` here. Additionally, all other places in code doing lookup by email will then be case-insensitive too. This would be an even better solution!
Huge thanks for pushing this forward!